bcgov/moh-prime
# PRIME-2050 remote users defect in site registration

[PRIME-2050](https://bcgovmoh.atlassian.net/jira/software/c/projects/PRIME/boards/1?modal=detail&selectedIssue=PRIME-2050)

**Remote Users now have a single certification instead of a list of certifications.**

### How to test:

1. Create a site
2. Add remote user
3. Click **Back** (you should see the "Unsaved Changes" warning dialogue)
4. Select **Keep Changes**
4. Click **Save and Continue** so you save that new remote user
5. Go back to Remote Users page

###
1. Click **Update Information** on the RU card
2. Click **Back** without changes (no dialogue)
3. In Remote Users page Click **Back** again (You should not see the dialogue)

###
1. Click update **Update Information**
2. Make changes
3. Click **Back**  (you should get the "Unsaved Changes" warning dialogue)
4. Discard Changes
5. **Remote Practitioner added/updated** should not be displayed
6. In Remote Users page Click **Back** again (You should not see the dialogue)

###
1. Click update **Update Information**
2. Make changes
3. Click **Continue** 
4. **Remote Practitioner added/updated** should not be displayed
5. In Remote Users page Click **Back** again (you should see the "Unsaved Changes" warning dialogue)

After all tests are good. Submit site.

Check mailhog and see the **Site Registration Review** document has the proper remote users and their details.
